engine/src/flutter/shell/platform/embedder/embedder.h

```diff
@@ -1693,6 +1693,11 @@ typedef struct {
   // The set of semantics flags associated with this node. Prefer to use this
   // over `flags__deprecated__`.
   FlutterSemanticsFlags* flags2;
+  /// An identifier for the semantics node in native accessibility hierarchy.
+  /// This value should not be exposed to the users of the app.
+  /// This is usually used for UI testing with tools that work by querying the
+  /// native accessibility, like UIAutomator, XCUITest, or Appium.
+  const char* identifier;
```

Comment:
Could you update the test that verifies this value gets sent to the embedder?

For example, this Dart code sets semantic nodes' tooltip value:

https://github.com/flutter/flutter/blob/7b8fceda91d843546d08768a311edaf240470aa1/engine/src/flutter/shell/platform/embedder/fixtures/main.dart#L195

This is verified here: https://github.com/search?q=repo%3Aflutter%2Fflutter+%22ASSERT_EQ%28std%3A%3Astrncmp%28kTooltip%2C+node-%3Etooltip%2C+sizeof%28kTooltip%29+-+1%29%22&type=code